bears/scss/SCSSLintBear.py

```diff
@@ -1,3 +1,5 @@
+import yaml
```

Comment:
yes, its needed, thats how im generating the specific requirements.txt :D we need it in REQUIREMENTS too